TMP2022/technical-writing-track/lisa-tw-funke/README.md

```diff
@@ -0,0 +1,499 @@
+4. Create your card PIN then tap Next.
+5. Type in a delivery address then tap **Next**.
+6. Confirm your delivery address.
+
+![physical card](https://downloads.intercomcdn.com/i/o/413774932/5566147488e4bfa0eb72a8db/K6.jpg)
+
+![physical card 2](https://downloads.intercomcdn.com/i/o/413774987/da8746c680d5f48fd61774d1/K8.jpg)
+
+**After receiving your card:**
+
+1. Open Kuda and tap **Cards** at the bottom of your dashboard.
+2. Tap **Activate Card**. 
+3. Read the activation instructions and tap **Activate Card**.
+4. Type in your card number (on the back of your card) then tap **Next.**
+
+![receive card](https://downloads.intercomcdn.com/i/o/421489109/a4888e1f8652be6ced3c1491/K3.jpg)
+
+![receive card 2](https://downloads.intercomcdn.com/i/o/421489159/59ea6738993c2a99ac99c4ea/K6.jpg)
+
+##### **Manage Cards**
+
+You can choose the channels (POS, ATM and online) where your Kuda Visa Card works and set a card transaction limit for each of those channels.
+
+**To manage your card:**
+
+1. Open Kuda.
+2. Tap **Cards**.
+3. Tap **Manage Cards**.
+4. Tap a transaction channel - **POS, ATM** or **Online**.
+5. Tap to turn off or turn on card transactions for the channel.
+6. You can also set a card transaction limit for the channel.
+
+![manage card](https://downloads.intercomcdn.com/i/o/471356564/d9bb99be5f4ff08b47f0fddd/K8.jpeg)
+
+![manage card 2](https://downloads.intercomcdn.com/i/o/471357205/f927a4624682155eed942484/K6.jpeg)
+
+
+## **Get Paid** <a name="get-paid"></a>
+
+Kuda offers a referral function. By utilizing your personal referral code, you can encourage friends and relatives to sign up for Kuda, and you will get compensated for it. 
+
+Everyone you recommend must sign up for Kuda using your referral link. Tap **More** at the bottom of your Kuda dashboard, then tap **Referrals** to see your code and share it.
+
+To get referral payments, you must verify your Bank Verification Number (BVN) and add a valid ID.
+
+Each individual you refer must add their BVN and a valid ID to their Kuda app and purchase at least 500 naira worth of airtime in order for you to get rewarded for the referral.
+
+You will receive 1,000 naira when you claim your referrals and 1,000 naira will be sent to each person who follows the referral instructions using your referral code.
+
+After the person you referred has finished the referral process, you will have 24 hours to claim your referral bonus.
+
+**To share your referral code:**
+
+1. Open Kuda and tap **More** at the bottom of your dashboard.
+2. Tap **Referrals**.
+3. Share your referral code.
+
+![referral](https://downloads.intercomcdn.com/i/o/513703742/ccc53118d920f095a9951819/K6.png)
+
+
+## Conclusion <a name="conclusion"></a>
+
+Visit [this site] (https://help.kuda.com/en/) for further details and inquiries about any Kuda services, goods, and so forth. You can contact their customer service department via email at help@kudabank.com or by calling the Kuda Bank contact number 09032200350.
```

Comment:
@Liio-24 you still need to correct the markdown in this link we talked about. Asides from that, awesome work!